packages/light-apps/src/globalStyle.tsx

```diff
@@ -22,3 +22,15 @@ export const GlobalStyle = createGlobalStyle`
     -moz-osx-font-smoothing: grayscale;
   }
 `;
+
+const substrateLightTheme = {
+  black: '#222',
+  cardBorder: '#f2f2f2',
+  grey: '#888',
+  lightBlue1: '#53a0fd',
```

Comment:
I think this is okay, but you're right that the hover is very subtle. Let's add a `darkBlue: '#5c53fc'` and use that for onHover instead, but keep the lightBlues as they work well for the gradient.